### ISSUE TITLE
Support bulk import/export from the local filesystem

### ISSUE THREAD

**Issue by lmsurpre** [IBM/FHIR#1055]
**Is your feature request related to a problem? Please describe.**
While less valid for true integration scenarios, sometimes its useful just to load some ndjson files from a local filesystem or a volume mount.

**Describe the solution you'd like**
Via configuration, it should be possible to:
1. Allow the server to import from file urls that start with `file://`
2. Export to files in a designated directory instead of exporting them to some S3-compatible object store

In fact, I'd like the local filesystem to be the "default" configuration so that we have a working import/export feature out-of-the box.

**Describe alternatives you've considered**
This is an alternative to the existing options that are already supported

**Additional context**
This would be especially handy for #1054

**Comment by prb112** [comment]
Let's confirm this work right now, I think it's technically supported

**Comment by lmsurpre** [comment]
Import/Export from the filesystem is now the default.

Whereas I was thinking that users could just pass a url that start with `file://` in their `$import` request, Paul opted to have them pass a separate parameter named `storageDetail`.
To signify an import from IBM COS, this should be set to `ibm-cos` (which is also the default I think).  For AWS S3 (or an S3-compatible provider) its `aws-s3`.
And to import from the filesystem, this parameter must be set to `file`.
In any case, this setting must match the actual configuration on the server.
I think it was done this way to be a bit more secure (we wouldn't want users being able to request us to load just any file from the server's filesystem...it makes sense to sandbox that to some configured directory). 
However, I think we'll need some additional user feedback on this feature to determine how well the current design is working.

My one complaint with this implementation is that it still isn't functional 'out-of-the-box'.
Specifically, because our default config sets `"fileBase": "/fhir-server/output"`, it requires users to either:
A. change this default setting in fhir-server-config.json; or
B. create this absolute directory for the output

I've opened https://github.com/IBM/FHIR/issues/2086 for this.


Finally, I noticed that the files created in a file-based export are placed directly in the configured fileBase directory with a filename that matches the random string that was generated for that particular export.  This is probably ok, but I think it might be better for each export to have its own subdirectory under the configured fileBase directory.  For this, I opened #2087
